docs/docs/genai/tracing/quickstart/index.mdx

```diff
@@ -0,0 +1,137 @@
+import ImageBox from '@site/src/components/ImageBox';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TabsWrapper from '@site/src/components/TabsWrapper';
+import ServerSetup from "@site/src/content/setup_server.mdx";
+
+# Tracing Quickstart
+
+This quickstart guide will walk you through setting up a simple GenAI application with MLflow Tracing. In less than 10 minutes, you'll enable tracing, run a basic application, and explore the generated traces in the MLflow UI.
+
+## Prerequisites
+
+Make sure you have started the MLflow server.
+If you don't have the MLflow server running yet, just follow these simple steps to get it started.
+
+<ServerSetup />
+
+## Create a MLflow Experiment
+
+The traces your GenAI application will send to the MLflow server are grouped into MLflow experiments. We recommend creating one experiment for each GenAI application.
+
+Let's create a new MLflow experiment using the MLflow UI so that you can start sending your traces.
+
+<ImageBox src="/images/llms/tracing/quickstart/mlflow-ui-new-experiment.png" alt="New Experiment"/>
+
+1. Navigate to the MLflow UI in your browser at [http://localhost:5000](http://localhost:5000).
+2. Click on the <div className="inline-flex rounded-sm bg-sky-700 px-2 py-1.5 text-sm font-semibold text-white">Create</div> button on the top right.
+3. Enter a name for the experiment and click on "Create".
+
+_You can leave the `Artifact Location` field blank for now. It is an advanced configuration to override where MLflow stores experiment data._
+
+## Dependency
+
+To connect your GenAI application to the MLflow server, you will need to install the MLflow client SDK.
+
+<TabsWrapper>
+<Tabs>
+<TabItem value="python" label="Python(OpenAI)" default>
+```bash
+pip install --upgrade mlflow openai>=1.0.0
+```
+</TabItem>
+
+<TabItem value="typescript" label="Typescript(OpenAI)" default>
+```bash
+npm install mlflow-openai
+```
+</TabItem>
+</Tabs>
+</TabsWrapper>
+
+:::info
+
+While this guide features an example using the OpenAI SDK, the same steps apply to other LLM providers, including Anthropic, Google, Bedrock, and many others.
+
+For a comprehensive list of LLM providers supported by MLflow, see the <ins>[LLM Integrations Overview](/genai/tracing/integrations)</ins>.
+
+:::
+
+## Start Tracing
+
+Once your experiment is created, you're ready to connect to the MLflow server and begin sending traces from your GenAI application.
+
+<TabsWrapper>
+<Tabs>
+<TabItem value="python" label="Python(OpenAI)" default>
+
+```python
+import mlflow
+from openai import OpenAI
+
+# Specify the tracking URI for the MLflow server.
+mlflow.set_tracking_uri("http://localhost:5000")
+
+# Specify the experiment you just created for your GenAI application.
+mlflow.set_experiment("My Application")
```

Comment:
I wonder whether setting env vars with MLFLOW_TRACKING_URI and MLFLOW_EXPERIMENT_ID is better 🤔 In that way we can avoid teaching new users to use 'set_experiment' from the beginning